Session/Open Groups/JoinOpenGroupVC.swift

```diff
@@ -166,8 +166,24 @@ final class JoinOpenGroupVC: BaseVC, UIPageViewControllerDataSource, UIPageViewC
                     Storage.shared.writeAsync { db in
                         try MessageSender.syncConfiguration(db, forceSyncNow: true).retainUntilComplete() // FIXME: It's probably cleaner to do this inside addOpenGroup(...)
                     }
+                    
+                    let maybeThread: SessionThread? = Storage.shared.write { db in
+                        let threadId: String = OpenGroup.idFor(roomToken: roomToken, server: server)
+                        return try SessionThread.fetchOrCreate(db, id: threadId, variant: .openGroup)
+                    }
```

Comment:
The `OpenGroupManager.shared.add` method should already be creating the thread so you should be able to remove this `write` call entirely